test/tools/ossfuzz/proto_to_yul.cpp

```diff
@@ -0,0 +1,234 @@
+{
+	switch (_x.op()) {
+		case BinaryOp::ADD:
+			_os << "add(" << _x.left() << "," << _x.right() << ")";
+			break;
+		case BinaryOp::SUB:
+			_os << "sub(" << _x.left() << "," << _x.right() << ")";
+			break;
+		case BinaryOp::MUL:
+			_os << "mul(" << _x.left() << "," << _x.right() << ")";
+			break;
+		case BinaryOp::DIV:
+			_os << "div(" << _x.left() << "," << _x.right() << ")";
+			break;
+		case BinaryOp::MOD:
+			_os << "mod(" << _x.left() << "," << _x.right() << ")";
+			break;
+		case BinaryOp::XOR:
+			_os << "xor(" << _x.left() << "," << _x.right() << ")";
+			break;
+		case BinaryOp::AND:
+			_os << "and(" << _x.left() << "," << _x.right() << ")";
+			break;
+		case BinaryOp::OR:
+			_os << "or(" << _x.left() << "," << _x.right() << ")";
+			break;
+		case BinaryOp::EQ:
+			_os << "eq(" << _x.left() << "," << _x.right() << ")";
+			break;
+		case BinaryOp::LT:
+			_os << "lt(" << _x.left() << "," << _x.right() << ")";
+			break;
+		case BinaryOp::GT:
+			_os << "gt(" << _x.left() << "," << _x.right() << ")";
+			break;
+	}
+	return _os;
+}
+
+std::ostream& operator<<(std::ostream& _os , const VarDecl& _x)
+{
+	_os << "let x_" << _x.id() << " := " << _x.expr() << "\n";
+	return _os;
+}
+
+std::ostream& operator<<(std::ostream& _os , const TypedVarDecl& _x)
+{
+	_os << "let x_" << _x.id();
+	switch (_x.type())
+	{
+		case TypedVarDecl::BOOL:
+			_os << ": bool := " << _x.expr() << " : bool\n";
+			break;
+		case TypedVarDecl::S8:
+			_os << ": s8 := " << _x.expr() << " : s8\n";
+			break;
+		case TypedVarDecl::S32:
+			_os << ": s32 := " << _x.expr() << " : s32\n";
+			break;
+		case TypedVarDecl::S64:
+			_os << ": s64 := " << _x.expr() << " : s64\n";
+			break;
+		case TypedVarDecl::S128:
+			_os << ": s128 := " << _x.expr() << " : s128\n";
+			break;
+		case TypedVarDecl::S256:
+			_os << ": s256 := " << _x.expr() << " : s256\n";
+			break;
+		case TypedVarDecl::U8:
+			_os << ": u8 := " << _x.expr() << " : u8\n";
+			break;
+		case TypedVarDecl::U32:
+			_os << ": u32 := " << _x.expr() << " : u32\n";
+			break;
+		case TypedVarDecl::U64:
+			_os << ": u64 := " << _x.expr() << " : u64\n";
+			break;
+		case TypedVarDecl::U128:
+			_os << ": u128 := " << _x.expr() << " : u128\n";
+			break;
+		case TypedVarDecl::U256:
+			_os << ": u256 := " << _x.expr() << " : u256\n";
+			break;
+	}
+	return _os;
+}
+
+std::ostream& operator<<(std::ostream& _os , const UnaryOp& _x)
+{
+	switch (_x.op()) {
+		case UnaryOp::NOT:
+			_os << "not(" << _x.operand() << ")";
+			break;
+		case UnaryOp::MLOAD:
+			_os << "mload(" << _x.operand() << ")";
+			break;
+		case UnaryOp::SLOAD:
+			_os << "sload(" << _x.operand() << ")";
+			break;
+	}
+	return _os;
+}
+
+std::ostream& operator<<(std::ostream& _os , const AssignmentStatement& _x)
+{
+	return _os  << _x.lvalue() << " := " << _x.rvalue() << "\n";
+}
+
+std::ostream& operator<<(std::ostream& _os , const IfStmt& _x)
+{
+	return _os  << "if " << _x.cond() << " {\n"
```

Comment:
I meant the following:
```
return _os <<
  "if " <<
  _x.cond() <<
  " {\n" <<
  _x.if_body() <<
  "}\n";
```
i.e. newline after each `<<`.

Reply:
thank you.